esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -2026,6 +2026,8 @@ void WsWuInfo::getWorkunitEclAgentLog(const char* processName, const char* fileN
     if (!isEmptyString(outFile))
     {
         outIOS.setown(createBufferedIOStreamFromFile(outFile, IFOcreate));
+        if(!outIOS)
+            throw MakeStringException(ECLWATCH_INTERNAL_ERROR, "Cannot open stream for file %s", outFile);
```

Comment:
MakeStringException should be deprecated, use makeStringException or makeStringExceptionV which explicitly say whether formatting or not should be used. 

The same for other MakeStringExceptions in this PR.